### PR TITLE
set minimum sector expiration to 180 days

### LINKED ISSUE #734: Sector Lifetime bounds
We need to change the enforced sector lifetime bounds.  The maximum sector lifetime is already enforced but the parameter needs to change to 366 days.

We are not yet enforcing sector minimum lifetime but need to do so at precommit time.  The minimum sector lifetime parameter is 180 days.

### PR DESCRIPTION
**Pull request by acruikshank**
closes #734

### Motivation

We need to enforce both a minimum and a maximum sector lifetime.

### Proposed Changes

1. Introduce `MinSectorExpiration` of `180*EpochsInDay` to miner policy.
2. Add check the sector meets minimum expiration criteria in miner actor `validateExpiration`.
3. Fix miner actor tests so that sectors that are supposed to be valid expire after at least 180 days.
4. Add test for minimum sector bound.

### CODE REVIEW

File: actors/builtin/miner/miner_test.go [hunk 1]
```diff
@@ -264,67 +266,71 @@ func TestCommitments(t *testing.T) {
...
 		rt.ExpectAbort(exitcode.ErrIllegalArgument, func() {
-			actor.preCommitSector(rt, actor.makePreCommit(oldSector.SectorNumber, challengeEpoch, deadline.PeriodEnd(), nil))
+			actor.preCommitSector(rt, actor.makePreCommit(oldSector.SectorNumber, challengeEpoch, expiration, nil))
 		})
 		rt.Reset()
 
 		// Bad sealed CID
 		rt.ExpectAbortConstainsMessage(exitcode.ErrIllegalArgument, "sealed CID had wrong prefix", func() {
-			pc := actor.makePreCommit(101, challengeEpoch, deadline.PeriodEnd(), nil)
+			pc := actor.makePreCommit(102, challengeEpoch, deadline.PeriodEnd(), nil)
```
Comment (ZenGround0) [inline actors/builtin/miner/miner_test.go:289]: Out of curiosity what motivated the change from 101 to 102?

Comment (acruikshank) [reply actors/builtin/miner/miner_test.go:289]: A lot of these failure tests were failing because they were duplicate precommits instead of the intended failure. The change to 102 fixes that.

File: actors/builtin/miner/policy.go [hunk 2]
```diff
@@ -108,10 +108,13 @@ var FaultMaxAge = WPoStProvingPeriod * 14
...
 const WorkerKeyChangeDelay = ChainFinality
 
+// Minimum number of epochs past the current epoch a sector may be set to expire.
+const MinSectorExpiration = 180 * builtin.EpochsInDay
+
 // Maximum number of epochs past the current epoch a sector may be set to expire.
 // The actual maximum extension will be the minimum of CurrEpoch + MaximumSectorExpirationExtension
 // and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
-const MaxSectorExpirationExtension = builtin.EpochsInYear
+const MaxSectorExpirationExtension = 366 * builtin.EpochsInDay
```
Comment (nicola) [inline actors/builtin/miner/policy.go:117]: why was this changed?

Comment (ZenGround0) [reply actors/builtin/miner/policy.go:117]: This is latest cryptoecon param recommendation.  cc @zixuanzh

Comment (acruikshank) [reply actors/builtin/miner/policy.go:117]: The difference is that EpochsInYear is 365*builtin.EpochsInDay and the new spec says 366 days (leap-year upper bound). I don't know if this is a change motivated by real concerns or just an inconsistency in how various teams define a year, but I thought it best to stick to the spec.
